src/lib.rs

```diff
@@ -79,6 +81,9 @@ pub fn build(target: &str) {
   cmd.arg("-C");
   cmd.arg(&gn_out_dir);
   cmd.arg(target);
+  for item in env {
+    cmd.env(item.0, item.1);
+  }
```

Comment:
Note that ninja is also called in src/deps.rs ... Probably you should change the `ninja()` function to take an env and gn_out_dir and return a `Command`